spikingjelly/activation_based/triton_kernel/spikezip_kernel.py

```
@@ -0,0 +1,226 @@
+    q_final = torch.empty_like(q)
+    acc_q_final = torch.empty_like(acc_q)
+    cur_output = torch.empty_like(q)
+
+    x_seq_flat = x_seq.reshape(T, N)
+    q_flat = q.reshape(N)
+    acc_q_flat = acc_q.reshape(N)
+    out_seq_flat = out_seq.reshape(T, N)
+    q_final_flat = q_final.reshape(N)
+    acc_q_final_flat = acc_q_final.reshape(N)
+    cur_output_flat = cur_output.reshape(N)
+    grid = lambda meta: (triton.cdiv(N, meta["BLOCK_N"]),)
```

Ruff (0.15.20)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `grid` as a `def`

(E731)